modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -35,6 +35,9 @@ Most of the scopes in the https://anypoint.mulesoft.com/accounts/api/v2/oauth2/.
 * `email`: Read-only access to the user's email.
 * `offline_access`: Access to the user's data when they are not logged in.
 
+[NOTE]
+Apps that use `client_credentials` grant type always have the `profile` scope assigned to them. The data returned by queries using the scope correspond to the user who created the connected app. 
```

Comment:
always have the profile scope assigned to them implicitly

Reply:
```suggestion
Apps that use `client_credentials` grant type always have the `profile` scope assigned to them implicitly. The data returned by queries using the scope correspond to the user who created the connected app. 
```